doc/man1/openssl-x509.pod.in

```diff
@@ -374,17 +378,33 @@ The serial number can be decimal or hex (if preceded by C<0x>).
 
 Set the serial to be one more than the number in the certificate.
 
+=item B<-notbefore> I<date>
+
+This allows the start date to be explicitly set. The format of the
+date is YYMMDDHHMMSSZ (the same as an ASN1 UTCTime structure), or
+YYYYMMDDHHMMSSZ (the same as an ASN1 GeneralizedTime structure). In
+both formats, seconds SS and timezone Z must be present.
+Cannot be used together with the B<-preserve_dates> option.
+
+=item B<-notafter> I<date>
+
+This allows the expiry date to be explicitly set. The format of the
+date is YYMMDDHHMMSSZ (the same as an ASN1 UTCTime structure), or
+YYYYMMDDHHMMSSZ (the same as an ASN1 GeneralizedTime structure). In
+both formats, seconds SS and timezone Z must be present.
+Cannot be used together with the B<-preserve_dates> option.
+
 =item B<-days> I<arg>
 
 Specifies the number of days until a newly generated certificate expires.
 The default is 30.
-Cannot be used together with the B<-preserve_dates> option.
+Cannot be used together with the options B<-preserve_dates> and B<-notafter>.
```

Comment:
This should probably state what this does if the notbefore is set.

Reply:
You're right, I missed that one.

Reply:
The current implementation of `set_cert_times` from `apps/lib/app.c` always uses "_now_" as base for the `-days` option, regardless of a given `startdate`.
Changing this behavior could have bigger impact.

Reply:
Yes.

Please also add to `-days` in `openssl-ca.pod.in`:
`Regardless of the option B<-startdate>, the days are always counted from today.`